json/intermediate-small.json

```diff
@@ -201,5 +201,12 @@
       ],
       "exclude_node_ids": []
     }
-  ]
+  ],
+  "metadata": {
+    "title": "3 GB",
+    "subtitle": "Small",
+    "size": 3,
```

Comment:
Looking at the plugin, this "size" is to compare with available disk space. We should take the oportunity to also rename it to something more explicit like "required_gigabytes".